UnitystationLauncher/Services/InstallationService.cs

```diff
@@ -421,23 +443,35 @@ private async Task StartDownloadAsync(Download download)
                 try
                 {
                     ZipArchive archive = new(progressStream);
-
-                    // TODO: Enable extraction cancelling
-                    archive.ExtractToDirectory(download.InstallPath, true);
-
-                    Log.Information("Download completed");
-
-                    _installations.Add(new()
+                    
+                    
+                    //TODO UI
+                    Action<string> info = new Action<string>((string log) => { });
```

Comment:
I'm not really sure what the purpose of these two `Action<string>`-s is, I see them being invoked in `CodeScanService.OnScan` but these logs and errors don't seem to be used for anything.

Reply:
the idea being as it would show up on the UI but idk how that would be implemented

Reply:
Ah alright, that should be fine to keep for now then. We can create an issue as a backlog item for displaying these.

Reply:
shall I make the issue now?